README.md

```diff
@@ -245,11 +245,17 @@ adb push /path/to/your/document.pdf /sdcard/document.pdf
 ```diff
 # Uncomment this line to define a global platform for your project
 -   # platform :ios, '9.0'
-+   platform :ios, '11.0'
++   platform :ios, '12.0'
 ...
+```
+Add the podspec link for `PSPDFKit` in `ios/Podfile` as shown below:
+
+__Optional:__ If you wish to use a specific version of `PSPDFKit` or the nightly version, you should use the appropriate podspec link.
```

Comment:
We should only support the latest version. For example, using an older version of PSPDFKit (say v8) will not work.

```suggestion
```